writer.go

```diff
@@ -73,6 +74,12 @@ type WriterConfig struct {
 	// The default is to use a target batch size of 100 messages.
 	BatchSize int
 
+	// Limit the maximum size of a request in bytes before being sent to
+	// a partition.
+	//
+	// The default is to use a kafka default value of 1048576.
+	MaxMessageBytes int
```

Comment:
Should this be called `BatchBytes` to fit the naming scheme of `BatchSize` and `BatchTimeout`?